CS410Assignment2/src/sensor/SensorApplication.java

```diff
@@ -0,0 +1,88 @@
+package sensor;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.GridLayout;
+import java.awt.Rectangle;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JProgressBar;
+import javax.swing.JTextField;
+import javax.swing.border.TitledBorder;
+import javax.swing.plaf.basic.BasicProgressBarUI;
+
+public class SensorApplication extends JFrame {
+	
+	
+	
+	public class BarUI extends BasicProgressBarUI {
+        private Rectangle r = new Rectangle();
+    }
+	
+	Color c;
+	JTextField textField; //
+	JLabel rvLabel;
+	String type;
+	String report;
+	String valueString;
+	double value;
+	int max;
+
+	
+	
+	
+
+	public SensorApplication() {
+		
+		setTitle("Sensor Tracker");
+		setLayout(new GridLayout(3, 1));
+		
+		SensorAdapter[] sensors = {new PressureSensorAdapter(new PressureSensor()),
+				new TemperatureSensorAdapter(new TemperatureSensor()), 
+				new RadiationSensorAdapter(new RadiationSensor())};
+		
+		for (SensorAdapter sensors1 : sensors) {
+            value= sensors1.getValue();

```

Comment:
Consistency in terms of indentation could be better for the for loop here. Should be tabbed in more, few lines are randomly tabbed. Spacing could also be condensed a little. Format issues are super minor though, no big deal.